lib/server/connect.js

```diff
@@ -40,6 +41,9 @@ Meteor.methods({
       data.times.push(ev);
     });
 
+    // set unique id for each server session
+    data._id = serverId;
```

Comment:
Don't add this as `_id`.
Add it as a field into the document as `serverId` along with `browserId` and `clientId`